src/views/EditorView.tsx

```diff
@@ -0,0 +1,185 @@
+import { Editor, EditorContent } from '@tiptap/react'
+
+const MenuBar = ({ editor }: { editor: Editor }) => {
+    if (!editor) {
```

Comment:
I don't think `editor` is nullable, so this can be removed.

Reply:
Thanks for the catch!